api_specifications/case-api-0.1.0.yaml

```
@@ -147,15 +147,15 @@ paths:
       summary: Search for cases
       description:  |
         This API deals with getting detailed information about a case, it will be access controlled. The basic premise will be that all validations are mandatory based on the logged in person’s individualID/authentication token, and this should not be sent as input parameter (as then a person can login and then request details by passing anyone else’s ID). In addition, search will always include tenancy as a parameter. Case details will be visible using the following logic.
-  
+        
```

yamllint

[error] trailing spaces

(trailing-spaces)